### ISSUE TITLE
Wired network result when training on COCO dataset.

### ISSUE THREAD

**Issue by Geekking** [fizyr/keras-retinanet#235]
Has anyone reproduce the performance provided by this repo?
I try to rerun the experiment on COCO dataset using the default configuration, lr 1e-5, 1000 step, 50 epoch, 
After training several(7,8 ) epoch, the network didn't predict any positive box on coco val2017 validation set. Could anybody provide debug idear? Thanks

**Comment by hgaiser** [comment]
I assume you meant 10000 step size? I'm not sure 7 or 8 epochs is enough. What is the loss that you're seeing at that time? Try letting it run for the 50 epochs.

ps. I don't remember exactly for how long the pretrained COCO network was trained for, but I believe it was more in the order of 100 epochs, and maybe even with a step size of 20000.

ps2. For further discussions on this topic, please join the Slack channel so that we can keep the issues page for implementation issues.

**Comment by hgaiser** [comment]
It's possible the issue with transform generators was messing with your results (fixed with https://github.com/fizyr/keras-retinanet/pull/240).
